tests/ci_install.sh

```diff
@@ -1,9 +1,31 @@
 set -x   # Show which command is being run
 
-echo "Travis build stage: $TRAVIS_BUILD_STAGE_NAME"
+case ${RUNNER_OS} in
+linux|Linux)
+    sudo apt-get install \
+      libhdf5-serial-dev \
+      libnetcdf-dev \
+      libproj-dev \
+      proj-data \
+      proj-bin \
+      libgeos-dev \
+      libopenmpi-dev
+    ;;
+osx|macOS)
```

Comment:
does there need to be a windows step here too?

Reply:
nope, it's covered below.